src/components/data-table-v2/_data-table-v2-sort.scss

```diff
@@ -66,3 +66,118 @@
     min-width: 14px;
   }
 }
+
+@mixin data-table-v2-sort--x {
+  // -------------------------------------
+  // Sortable table
+  // -------------------------------------
+
+  .#{$prefix}--data-table-v2--sort th:hover {
+    background: $data-table-column-hover;
+    border-top: 1px solid $data-table-column-hover;
+    border-bottom: 1px solid $data-table-column-hover;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-icon:hover,
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-menu:hover,
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-expand-v2:hover {
+    background: $ui-03;
+    border-top: 1px solid $ui-03;
+    border-bottom: 1px solid $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2--sort th {
+    padding: 0;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-checkbox {
+    padding: rem(10px) $spacing-05;
+  }
+
+  // -------------------------------------
+  // Th > Button
+  // -------------------------------------
+  .#{$prefix}--table-sort-v2 {
+    @include button-reset(false);
+    position: relative;
+    font: inherit;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    color: $text-01;
+    padding: 0 $spacing-04;
+  }
+
+  .#{$prefix}--table-sort-v2:focus {
+    outline: 0;
```

Comment:
Seems like we might need this focus outline for the `button` control, right?

Reply:
I didn't include a focus outline based on a discussion I had with design on slack. Because buttons aren't fluid like the `th`, when there's one column that has text longer than other, it looks weird. Like this: <img width="604" alt="Screen Shot 2019-03-21 at 4 30 09 PM" src="https://user-images.githubusercontent.com/32556167/54787073-a42e9680-4bf8-11e9-8e17-2a1797d751ea.png">

Reply:
@jnm2377 isn't the focus outline required for accessibility for the focus state on the button?

Reply:
cc: @dakahn question about a11y above

Reply:
the original issue was that the focus outline didn't cover the whole cell right? I guess we can set a fixed value for the height of the `<td>` (double check with design I guess), which would allow us to make buttons fill the height of the table cell. and then use ellipsis+tooltip for overflow content

Reply:
Some kind of outline/color differentiation with enough contrast is needed yeah

Reply:
We changed the overflow to be ellipsis so I added a focus outline. 👍